### PR TITLE
Optionally include external version in websocket connections

### PR DESCRIPTION
**Pull request by stkem**
@andrewconner @ebfaed @2is10 objections?

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/AuthenticatedWebSocketsController.scala [hunk 1]
```diff
@@ -143,11 +143,11 @@ trait AuthenticatedWebSocketsController extends ElizaServiceController {
   }
 
 
-  def websocket() = WebSocket.async[JsArray] { implicit request =>
+  def websocket(versionOpt: Option[String]) = WebSocket.async[JsArray] { implicit request =>
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/AuthenticatedWebSocketsController.scala:146]: May be useful to use the same variable name here as in the routes (`version`).

**Comment by andrewconner** [comment]
Nice

File: kifi-backend/modules/eliza/app/com/keepit/eliza/AuthenticatedWebSocketsController.scala (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/AuthenticatedWebSocketsController.scala:146]: good point

**Comment by 2is10** [comment]
:+1:

**Pull request merged**